app/config/index.js

```diff
@@ -131,6 +135,10 @@ const config = {
   endemics: {
     enabled: process.env.ENDEMICS_ENABLED
   },
+  multiSpecies: {
+    enabled: process.env.MULTI_SPECIES_ENABLED === 'true',
+    releaseDate: process.env.MULTI_SPECIES_RELEASE_DATE || '3000-01-01'
```

Comment:
I believe Busted have been to this year, and apparently it wasn't all that. 

Not convinced having a default here makes any sense. It's marked as required in the validation, but if we mess up configuring the application and don't specify it, then the default will be used and the banner will never show instead of failing fast and making us immediately aware we should be fixing the config to add the actual date.

Reply:
Removed, app will crash if it's not provided.